### PR TITLE
Currency iso locale dev

### LINKED ISSUE #663: Locale not respected for business.currency_iso_code()
# Bug report

## What's wrong

I'm trying to use mimesis to generate mock data for tests I'm going to develop and am writing [schema](https://mimesis.readthedocs.io/getting_started.html#schema-and-fields) to do so.

I set the locale when instantiating an object of class `Field()`, but on calling `business.currency_iso_code()` I get random ISO currency codes, whilst I was expecting (or hoping!) to get those for the current locale.

    import random
    from mimesis.schema import Field, Schema

    class SlotData2():
        def __init__(self, locale, max_currency=10):
            self.field = Field(locale)
            self.max_currency = max_currency

        def _generate_cost(self):
            cost = (
                lambda: {
                    'amount': self.field('business.price'),
                    'currency': self.field('business.currency_iso_code'),
                }
            )
            cost_schema = Schema(schema=cost)
            return cost_schema.create(random.randint(1, self.max_currency))

The `business.price` is always the correct locale, but the `business.currency_iso_code` varies as the following shows...

    In [200]: test2 = SlotData2('en-gb', 10)
    
    In [201]: test2._generate_cost()
    Out[201]: 
    [{'amount': '196.66 £', 'currency': 'BZD'},
     {'amount': '223.12 £', 'currency': 'VUV'}]

    In [202]: test2._generate_cost()
    Out[202]: 
    [{'amount': '626.08 £', 'currency': 'XCD'},
     {'amount': '904.78 £', 'currency': 'BOV'},
     {'amount': '836.96 £', 'currency': 'XUA'},
     {'amount': '529.19 £', 'currency': 'LYD'},
     {'amount': '341.14 £', 'currency': 'ILS'},
     {'amount': '136.68 £', 'currency': 'MGA'},
     {'amount': '691.84 £', 'currency': 'KHR'},
     {'amount': '493.02 £', 'currency': 'SDG'},
     {'amount': '250.04 £', 'currency': 'BGN'},
     {'amount': '649.36 £', 'currency': 'AUD'}]
    
    In [203]: test2._generate_cost()
    Out[203]: 
    [{'amount': '963.86 £', 'currency': 'MNT'},
     {'amount': '707.13 £', 'currency': 'TRY'},
     {'amount': '322.74 £', 'currency': 'SHP'}]

I checked with `Generic('en-gb')` and it too returns random ISO currency codes rather than the expected `GBP`...

    In [204]: g = Generic('en-gb')
    
    In [205]: g.business.currency_iso_code()
    Out[205]: 'AFN'
    
    In [206]: g.business.currency_iso_code()
    Out[206]: 'USD'
    
    In [207]: g.business.currency_iso_code()
    Out[207]: 'ALL'

## How is that should be

I would expect `business.currency_iso_code()` to return the code for the specified locale.

## System information

    python --version
    Python 3.6.7
    In [209]: mimesis.__version__
    Out[209]: '3.1.0'

### PR DESCRIPTION
**Pull request by ceccoemi**
# I have made things!

## Checklist

- [x] I'm sure that I did not unrelated changes in this pull request
- [x] I have created at least one test case for the changes I have made
- [x] I have updated the documentation for the changes I have made
- [x] I have added my changes to the [CHANGELOG.rst](https://github.com/lk-geimfari/mimesis/blob/master/CHANGELOG.rst)

## Related issues

- Closes #663 
- Refs #663 

Now the function `currency_iso_code()` from `Business()` provider returns the code relative of the current locale:
```
>>> p = p.mimesis.Business(locale='en')
>>> p.currency_iso_code()
'USD'
>>> p.currency_iso_code()
'USD'
>>> p.currency_iso_code()
'USD'
```

If you want to generate a random currency code (also from other country besides the supported locales) you need to add the parameter `allow_random=True`: 
```
>>> p = p.mimesis.Business(locale='en')
>>> p.currency_iso_code(allow_random=True)
'ILS'
>>> p.currency_iso_code(allow_random=True)
'BDT'
>>> p.currency_iso_code(allow_random=True)
'SEK'
```

### CODE REVIEW

**Comment by lk-geimfari** [comment]
@ceccoemi Sorry, my bad. Seems like everything is okay!

File: tests/test_providers/test_business.py [hunk 1]
```diff
@@ -28,7 +28,11 @@ def test_copyright(self, business):
         assert '©' in result
 
     def test_currency_iso_code(self, _business):
```
Comment (lk-geimfari) [inline tests/test_providers/test_business.py]: So, you need to use business fixture here to cover all locales.

Comment (lk-geimfari) [reply tests/test_providers/test_business.py]: @ceccoemi You can find such fixture [here](https://github.com/lk-geimfari/mimesis/blob/c3ea6ae31659bbba5502939ca3130b0c63fca744/tests/conftest.py#L26)

**Comment by lk-geimfari** [comment]
@ceccoemi Check out comments, please. Thanks!

File: tests/test_providers/test_business.py (hunk 1 above)
Comment (ceccoemi) [reply tests/test_providers/test_business.py]: @lk-geimfari I fixed the fixture! Please see my last commit.

**Pull request merged**